.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+<!--
+Adding a new resource or datasource? Use this checklist to get started: https://github.com/hashicorp/terraform-provider-hcp/blob/main/contributing/checklist-resource.md
+-->
+
+### :hammer_and_wrench: Description
+
+<!-- What code changed, and why? -->
+
+### :building_construction: Testing
+
+<!-- Include any feature flags that are required for testing. -->
```

Comment:
What is this section for exactly? For reporting acceptance test output? For demonstrating how to test changes locally? I would add more detail to the comment.

Reply:
Added more detail here based off of the Kubernetes provider template